src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -311,6 +311,12 @@ const AppWrapper = (props: { children: ReactNode }) => {
     let userName: { userName: string } = { userName: "" };
     const passcodeIsSet = await checkKeyStore(KeyStoreKeys.APP_PASSCODE);
     const seedPhraseIsSet = await checkKeyStore(KeyStoreKeys.SIGNIFY_BRAN);
+    const recoveryWalletProgress = await checkKeyStore(
```

Comment:
I've noticed an issue, not sure if this is related: when you select to either create a new account or recover an existing one and after that you reload the app, you cannot go back.

Here I took 2 videos, one for each of the choices the user can make, please take a look.


https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/1596f177-4c23-43bc-8f0d-b2b858fa585d

https://github.com/cardano-foundation/cf-identity-wallet/assets/119612231/d81cfdaa-8f3e-46ef-99ed-ac18343c834d

Reply:
I fixed. Please help me check.

https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/15d3130e-7809-4166-8950-a980a889fe49